### PR TITLE
removing confusing information from introduction

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* removing confusing information from introduction on plugin pages

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by JamesToohey — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

File: plugins/opensource/directionality.md [hunk 1]
```diff
@@ -12,10 +12,6 @@ controls: toolbar button
 
 This plugin adds directionality controls to the toolbar, enabling {{site.productname}} to better handle languages written from right to left. It also adds a toolbar button for each of its values, `ltr` for left-to-right text and `rtl` for right-to-left text.
 
-**Type:** `String`
-
-**Possible Values:** `ltr`, `rtl`
-
```
Comment (EkimChau) [inline plugins/opensource/directionality.md:18]: For this case would it be clearer to state

**Property** `toolbar`
**Type:** `String`	
**Possible Values:** `ltr`, `rtl`

having the property omitted is the confusing part as u need to imply it to toolbar

**Review by EkimChau — APPROVED** [review]

File: plugins/opensource/directionality.md (hunk 1 above)
Comment (lnewson) [reply plugins/opensource/directionality.md:18]: That part is covered in the next section I believe: https://www.tiny.cloud/docs/plugins/opensource/directionality/#toolbarbuttons

File: plugins/opensource/pagebreak.md [hunk 2]
```diff
@@ -14,10 +14,6 @@ This plugin adds page break support and enables a user to insert page breaks in
 
 It also adds a toolbar button and a menu item `Page break` under the `Insert` menu dropdown.
 
-**Type:** `String`
-
-**Default Value:** `"<!-- pagebreak -->"`
```
Comment (lnewson) [inline plugins/opensource/pagebreak.md:19]: This probably needs to be moved to the `pagebreak_separator` setting instead of just being deleted.

**Review by lnewson — APPROVED** [review]

File: plugins/opensource/pagebreak.md (hunk 2 above)
Comment (tylerkelly13) [reply plugins/opensource/pagebreak.md:19]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1838/commits/d611ea2b482f79339c7f50769540e80a8936e516

**Pull request merged**